OWL2/StaticAnalysis.hs

```diff
@@ -61,13 +61,13 @@ checkEntity s t@(Entity _ ty e) =
 
 -- | takes an iri and finds out what entities it belongs to
 correctEntity :: Sign -> IRI -> [Entity]
-correctEntity s iri =
-    [mkEntity AnnotationProperty iri | Set.member iri (annotationRoles s)] ++
-    [mkEntity Class iri | Set.member iri (concepts s)] ++
-    [mkEntity ObjectProperty iri | Set.member iri (objectProperties s)] ++
-    [mkEntity DataProperty iri | Set.member iri (dataProperties s)] ++
-    [mkEntity Datatype iri | Set.member iri (datatypes s)] ++
-    [mkEntity NamedIndividual iri | Set.member iri (individuals s)]
+correctEntity s iRi =
```

Comment:
Why `iRi` instead of `iri` (throughout the file)?

Reply:
Same as above.